internal/hcsoci/devices.go

```diff
@@ -47,6 +47,20 @@ func getDeviceExtensionPaths(annots map[string]string) ([]string, error) {
 	return extensions, nil
 }
 
+// getGPUVHDPath gets the gpu vhd path from the shim options or uses the default if no
+// shim option is set. Right now we only support Nvidia gpus, so this will default to
+// a gpu vhd with nvidia files
+func getGPUVHDPath(annot map[string]string) (string, error) {
+	gpuVHDPath, ok := annot[annotations.GPUVHDPath]
+	if !ok || gpuVHDPath == "" {
+		return "", fmt.Errorf("no gpu vhd specified %s", gpuVHDPath)
```

Comment:
doesn't look like the value of `gpuVHDPath` is useful at all here, since it's just an empty string.